ztp/kube-compare-reference/optional/ptp-config/PtpConfigDualCardGmWpc.yaml

```diff
@@ -0,0 +1,240 @@
+# The grandmaster profile is provided for testing only
+# It is not installed on production clusters
+# In this example two cards $iface_nic1 and $iface_nic2 are connected via
+# SMA1 ports by a cable and $iface_nic2 receives 1PPS signals from $iface_nic1
+apiVersion: ptp.openshift.io/v1
+kind: PtpConfig
+metadata:
+  name: grandmaster
+  namespace: openshift-ptp
+  annotations:
+    ran.openshift.io/ztp-deploy-wave: "10"
+spec:
+  profile:
+  - name: "grandmaster"
+    ptp4lOpts: "-2 --summary_interval -4"
+    phc2sysOpts: -r -u 0 -m -w -N 8 -R 16 -s $iface_nic1 -n 24
+    ptpSchedulingPolicy: SCHED_FIFO
+    ptpSchedulingPriority: 10
+    ptpSettings:
+      logReduce: "true"
+    plugins:
+      e810:
+        enableDefaultConfig: false
+        settings:
+          LocalMaxHoldoverOffSet: 1500
+          LocalHoldoverTimeout: 14400
+          MaxInSpecOffset: 100
+        pins: $e810_pins
```

Comment:
User will provide this content to fill in interface names. Ideally we would verify these values (anything for the NIC name, but the values should be as specified).

Reply:
updated to be user variable